src/coreclr/nativeaot/Runtime/windows/CoffNativeCodeManager.cpp

```diff
@@ -246,9 +246,8 @@ static_assert(sizeof(CoffNativeMethodInfo) <= sizeof(MethodInfo), "CoffNativeMet
 bool CoffNativeCodeManager::FindMethodInfo(PTR_VOID        ControlPC,
                                            MethodInfo *    pMethodInfoOut)
 {
-    // Stackwalker may call this with ControlPC that does not belong to this code manager
-    if (dac_cast<TADDR>(ControlPC) < dac_cast<TADDR>(m_pvManagedCodeStartRange) ||
-        dac_cast<TADDR>(m_pvManagedCodeStartRange) + m_cbManagedCodeRange <= dac_cast<TADDR>(ControlPC))
+    // Stackwalker may call this with ControlPC that does not managed
+    if (!IsManaged(ControlPC))
```

Comment:
Is this a virtual call now?

Reply:
This is c++, so it might be a direct call, since nothing derives from CoffNativeCodeManager.
It does not have to be virtual though. This can be factored out into something like `NativeCodeManagerBase`. 

There is some other functionality that does not seem to differ between Windows and Unix. I was not sure if we wanted a base class for that.